test/verify/check-system-s4u-ssh

```diff
@@ -90,8 +94,35 @@ gssapi.creds.rcred_cred_store.store_cred_into({ 'ccache'.encode(): 'FILE:/etc/co
         # enable gssapiauth
         sshd_machine.execute("sed -ri 's/#GSSAPIAuthentication (yes|no)/GSSAPIAuthentication yes/' /etc/ssh/sshd_config")
         sshd_machine.execute("systemctl daemon-reload && systemctl restart sshd")
+        client_machine.execute("KRB5RCACHEDIR=\"/var/cache/krb5rcache\" KRB5_TRACE=\"/dev/stderr\" KRB5CCNAME=\"/etc/cockpit.ccache\" ssh -vvv -K  user@sshserver.cockpit.lan echo hello")
 
-        client_machine.execute("KRB5_TRACE=\"/dev/stderr\" KRB5CCNAME=\"/etc/cockpit.ccache\" ssh -vvv -K  user@sshserver.cockpit.lan echo hello")
+        # configure gssapi authentication for sudo
+        sshd_machine.write('/etc/sudoers.d/cockpit-krb5-sudo', """
+Defaults    env_keep += "KRB5CCNAME"
```

Comment:
This is rather non-obvious. I wonder how we are going to deal with that for the real implementation.. But that is a discussion for another day.